src/main/java/seedu/address/logic/commands/allocatestudenttoteamcommands/AllocateStudentToTeamByIndexCommand.java

```diff
@@ -0,0 +1,117 @@
+package seedu.address.logic.commands.allocatestudenttoteamcommands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INDEX;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MODULECODE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TEAMNAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TUTORIALCLASS;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.CollectionUtil;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.module.ModuleCode;
+import seedu.address.model.module.TutorialClass;
+import seedu.address.model.module.TutorialTeam;
+import seedu.address.model.person.Person;
+
+/**
+ * Allocates a student to a team in a tutorial Class in TAHelper.
+ */
+public class AllocateStudentToTeamByIndexCommand extends AllocateStudentToTeamCommand {
+
+    public static final String COMMAND_WORD = "/allocate_team";
```

Comment:
I think there are certain variables which can be removed as they have already been defined in the parent abstract class